common/lib/dependabot/package_manager.rb

```diff
@@ -58,18 +58,36 @@ def deprecated?
       deprecated_versions.include?(version)
     end
 
+    sig { void }
+    def raise_if_unsupported!
+      return unless unsupported?
+
+      # "7.*, 8.*"
+      supported_versions_message = supported_versions.map do |v|
+        if v.to_s.include?("*")
+          v.to_s
+        else
+          "v#{v}.*"
+        end
+      end.join(", ")
+
+      raise ToolVersionNotSupported.new(name, version.to_s, supported_versions_message)
+    end
+
```

Comment:
We are dynamically creating supported version string here that is going to be shown in the error.